docs/source/programming_resources/shared/control_system_intro/The-FTC-Control-System.rst

```diff
@@ -0,0 +1,124 @@
+FTC Control System Introduction
```

Comment:
This is a duplicate to the "[About the FIRST Tech Challenge](https://first-tech-challenge-ftcdocs--11.com.readthedocs.build/en/11/overview/ftccontrolsystem.html)" page seen at the top of the main TOC. In addition this conflicts with "[FTC Control System Overview](https://first-tech-challenge-ftcdocs--11.com.readthedocs.build/en/11/stubs/ftc_control_system_overview.htm)" page.

Reply:
Yeah, we need a better "About FIRST Tech Challenge" page, I didn't replace it because I don't have anything better yet. I'll submit an issue for it.

Reply:
Issue Created, https://github.com/FIRST-Tech-Challenge/ftcdocs/issues/12

Reply:
Sorry, I totally didn't realize you pointed to the stub with respect to the conflict. I changed the main TOC to point to the real FTC Control System Overview page, and deleted the stub. Still need issue #12 .